doc/releases/release-notes-3.7.rst

```diff
@@ -215,32 +215,40 @@ Bluetooth
 * HCI Driver
 
   * Added support for Ambiq Apollo3 Blue series.
+  * Added support for NXP platforms.
 
 Boards & SoC Support
 ********************
 
 * Added support for these SoC series:
 
   * Added support for Ambiq Apollo3 Blue and Apollo3 Blue Plus SoC series.
+  * Added support for NXP mke15z7, mke17z7, mke17z9, MCXNx4x, RW61x
 
 * Made these changes in other SoC series:
 
   * ITE: Rename the Kconfig symbol for all ITE SoC variants.
-
+  * NXP IMX8M: added resource domain controller support
+  * NXP s32k146: set RTC clock source to internal oscillator
+    
 * Added support for these ARM boards:
 
   * Added support for :ref:`Ambiq Apollo3 Blue board <apollo3_evb>`: ``apollo3_evb``.
   * Added support for :ref:`Ambiq Apollo3 Blue Plus board <apollo3p_evb>`: ``apollo3p_evb``.
   * Added support for :ref:`Raspberry Pi 5 board <rpi_5>`: ``rpi_5``.
   * Added support for :ref:`Seeed Studio XIAO RP2040 board <xiao_rp2040>`: ``xiao_rp2040``.
   * Added support for :ref:`Mikroe RA4M1 Clicker board <mikroe_clicker_ra4m1>`: ``mikroe_clicker_ra4m1``.
-  * Added support for :ref:`Arduino UNO R4 WiFi board <arduino_uno_r4>`: ``arduino_uno_r4_wifi``.
-  * Added support for :ref:`Renesas EK-RA8M1 board <ek_ra8m1>`: ``ek_ra8m1``.
```

Comment:
why have you removed the renesas comment?

Reply:
That was unintended of course